Content.Shared/Speech/Components/MeleeSpeechComponent.cs

```diff
@@ -7,15 +8,28 @@ namespace Content.Shared.Speech.Components;
 [AutoGenerateComponentState]
 public sealed partial class MeleeSpeechComponent : Component
 {
+    [ViewVariables]
+    public EntityUid? User;
 
-	[ViewVariables(VVAccess.ReadWrite)]
+    [ViewVariables(VVAccess.ReadWrite)]
 	[DataField("Battlecry")]
 	[AutoNetworkedField]
 	public string? Battlecry;
 
     [ViewVariables(VVAccess.ReadWrite)]
```

Comment:
Don't mix tabs and spaces also please document your code, there's nothing on the system or component that explains what it actually does.